code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -312,12 +312,12 @@
 	loot = list(
 				/obj/item/ai_module/core/full/corp,
 				/obj/item/ai_module/core/full/maintain,
-				/obj/item/ai_module/core/full/drone,
 				/obj/item/ai_module/core/full/peacekeeper,
 				/obj/item/ai_module/core/full/reporter,
 				/obj/item/ai_module/core/full/robocop,
 				/obj/item/ai_module/core/full/liveandletlive,
-				/obj/item/ai_module/core/full/hulkamania
+				/obj/item/ai_module/core/full/hulkamania,
+				/obj/item/ai_module/reset/purge //under silicon policy, purged AIs can't randomly murder people for no good reason
```

Comment:
revert

Reply:
why

Reply:
Like, I'm pretty sure that the only place that you're gonna find these two pools in is in the AI upload, anyway. Don't you think that a purge board is more appropriate to be in the pool of boards that the RD has default access to (IIRC) than a "fuck over the AI and all of their slaved cyborgs until someone else mercy-resets them" board?

Reply:
admins being dumb with purge is not my problem.

naughty modules are sometimes locked more securely than the normal ones, and I dont want purge more easily accessible.

The relative shitlery doesnt matter, otherwise every board that isn't drone could make that case.

Reply:
I'm coding for the rules setup that we currently have, not one that we'd like to have but have no power to change.

Currently, the purge board is actually a pretty safe one. If the policy for purged AIs changes in the future such that purged AIs get free antag or something, I can move it back into the "dangerous" section.

Reply:
It is still dangerous in the sense that freeform has way more control in the sense it is more difficult to nuance the AI when you are having to work with a current lawset than it is to start from scratch.

Reset already exists and purge should be kept behind the additional security, admin implications or not.

Reply:
fucking hell, FINE